fury/actor.py

```diff
@@ -516,6 +521,21 @@ def line(lines, colors=None, opacity=1, linewidth=1,
     poly_mapper.SelectColorArray("Colors")
     poly_mapper.Update()
 
+    if depth_cue:
+        geom_shader_file = open("fury/shaders/line.geom", "r")
```

Comment:
This relative path breaks when the current working folder is not the root of `fury`. See PR https://github.com/dmreagan/fury/pull/1.